docs/Actions.md

```diff
@@ -23,6 +23,7 @@ import './path/to/other/Fastfile'
 - [Using git](#using-git)
 - [Using mercurial](#using-mercurial)
 - [Notifications](#notifications)
+- [Cordova](#cordova)
```

Comment:
This shouldn't be its own section, as it's only one action. Instead put it into Misc.